gitlabform/processors/project/members_processor.py

```diff
@@ -101,11 +102,25 @@ def _process_users(
         enforce_members: bool,
         keep_bots: bool,
     ):
+
+        project: Project = self.gl.projects.get(project_and_group)
```

Comment:
is there a reason not using the cached method here?

Reply:
Mostly that I wasn't fully aware of it 😁 I've stepped away for a bit, but will pull in the cached method instead, as that would definitely be better!

Reply:
Updated :)